views/live.ejs

```diff
@@ -0,0 +1,236 @@
+							<div class="modal-content">
+								<div class="modal-header">
+									<h4 class="modal-title">Payment</h4>
+									<button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+								</div>
+									<input type="hidden" name="eventId" value="<%=eventId%>" />
+									<input type="hidden" name="paymentAmount" id="paymentAmount" value=0 />
+									<div class="modal-body">
+										<div class="form-group">
+											<label for="donationValue">Donation Amount</label>
+											<input type="number" class="form-control" step="0.01" name="donationValue" min="1" max="9999999" required/>
+										</div>
+										<div class="form-group">
+											<label for="cardNo">Card Number</label>
+											<input type="number" class="form-control" name="cardNo" min="0" max="9999999999999999" required/>
+										</div>
+										<div class="form-group">
+											<label for="cvv" class="col-form-label">CVV</label>
+											<input type="number" class="form-control" name="cvv" min="0" max="999" required/>
+										</div>
+										<div class="form-group">
+											<label for="billingName" class="col-form-label">Card Holder Name</label>
+											<input type="text" class="form-control" name="billingName" required/>
+										</div>
+									</div>
+									<div class="modal-footer">
+										<button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
+										<button type="submit" class="btn btn-primary">Confirm</button>
+									</div>
+								</form>
+							</div>
+						</div>
+					</div>
+				
+		</div>	
+	</body>
+	<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.6/jquery.min.js" type="text/javascript"></script>
+	<script src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8/jquery-ui.min.js" type="text/javascript"></script>
+	<link href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8/themes/base/jquery-ui.css" rel="Stylesheet" type="text/css" />
+	<script> $("#datepicker").datepicker(); </script>
+	<script>
+	
+	var amountLoop = setInterval(function(){
+		$.ajax({
+			url: "amount/<%=eventId%>",
+			type: "POST",			
+			success: function(data) {
+			  $('#totalRaised').html(data);	
+			},
+			error:   function(xhr, text, err) {
+			}
+		  });
+	}, 2000);
+
+	var milestones = $('.milestones');
+
+	var achievedLoop = setInterval(function(){
```

Comment:
Do both of these need to loop every 2/1 seconds? I get one needs to poll frequently, but surely if the donation amount changes then it can just call the achievedLoop

Reply:
Would actually be more complicated because of the page refresh when people donate I'd need to keep track of previous value before donation. Not worth the time.